### PR TITLE
Fixes for runtime/compiler errors. 

### PR DESCRIPTION
**Pull request by ThorBl**
PR contains fixes for compiler/runtime errors. 
Affected files are:
-MEMCPY-OMPTarget.cpp
-REDUCE_STRUCT-OMPTarget.cpp

### CODE REVIEW

File: src/algorithm/MEMCPY-OMPTarget.cpp [hunk 1]
```diff
@@ -29,61 +29,63 @@ namespace algorithm
...
+  deallocOpenMPDeviceData(x, did);                                             \
+  deallocOpenMPDeviceData(y, did);
 
 
 void MEMCPY::runOpenMPTargetVariant(VariantID vid, size_t RAJAPERF_UNUSED_ARG(tune_idx))
 {
-  const Index_type run_reps = getRunReps();
-  const Index_type ibegin = 0;
-  const Index_type iend = getActualProblemSize();
+    const Index_type run_reps = getRunReps();
```
Comment (rhornung67) [inline src/algorithm/MEMCPY-OMPTarget.cpp:43]: Please don't change the format of the code unless it improves readability. Our convention is indent two-spaces, no tabs in source code.

Comment (ThorBl) [reply src/algorithm/MEMCPY-OMPTarget.cpp:43]: Yes, sorry. I have undone the formatting and will upload 'cleaned' files. That's why I marked the PR as draft.

**Comment by rhornung67** [comment]
@ThorBl I pulled the branch on your fork into our repo so all our CI checks will run. That PR will be merged when approved and checks pass. Here's the link: https://github.com/LLNL/RAJAPerf/pull/276

Discussion of your PR, if any, will continue on that PR.

**Review by rhornung67 — APPROVED** [review]
@ThorBl thanks for the fixes.  These kernels are relatively new and obviously haven't been checked with an OpenMP target offload build.

**Comment by rhornung67** [comment]
Changes in This PR were merged into develop: https://github.com/LLNL/RAJAPerf/pull/276

**Pull request closed without merging**